change-notes/1.18/analysis-javascript.md

```diff
@@ -81,6 +83,7 @@
 | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Arguments redefined | Fewer results | This rule previously also flagged redefinitions of `eval`. This was an oversight that is now fixed. |
+| Comparison between inconvertible types | Fewer results | This rule now flags fewer comparisons involving IIFE parameters. |
```

Comment:
Here and below: while it's technically true that the changes only affect IIFEs, it may be simpler to just say "function parameters/return values".